<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>diem_input</class>
 <widget class="QWidget" name="diem_input">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1146</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="BangDiem_Widget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>791</width>
     <height>421</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>281</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>BẢNG NHẬP ĐIỂM</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>561</width>
     <height>68</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="maximumSize">
       <size>
        <width>71</width>
        <height>16</height>
       </size>
      </property>
      <property name="text">
       <string>Niên khóa</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLineEdit" name="lineEdit__TenMon">
      <property name="placeholderText">
       <string>Chữ cái</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit__Hocky">
      <property name="placeholderText">
       <string>Số thường</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Học kỳ</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLineEdit" name="lineEdit__NienKhoa">
      <property name="placeholderText">
       <string>xxxx-xxxx</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Nhóm</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="lineEdit__Nhom">
      <property name="placeholderText">
       <string>Số Thường</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Tên môn</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_TraBangDiem">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>90</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Tra Bảng Điểm</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>811</width>
     <height>681</height>
    </rect>
   </property>
  </widget>
  <zorder>tableView</zorder>
  <zorder>BangDiem_Widget</zorder>
  <zorder>label_5</zorder>
  <zorder>gridLayoutWidget</zorder>
  <zorder>pushButton_TraBangDiem</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
